source/Command.cpp

```diff
@@ -272,6 +273,7 @@ void Command::Load(const DataNode &node)
 			{"scan", Command::SCAN},
 			{"jump", Command::JUMP},
 			{"mouseturninghold", Command::MOUSE_TURNING_HOLD},
+			{"aimturrethold", Command::AIM_TURRET_HOLD},
```

Comment:
```suggestion
			{"mouse turning hold", Command::MOUSE_TURNING_HOLD},
			{"aim turret hold", Command::AIM_TURRET_HOLD},
```
This is the correct formatting, it was fixed in Hurl's hold fire PR, and then broken again in the Great Unrefactor. ~~but it's probably not used anywhere, so I don't care much~~